### PR TITLE
Add "metadataIxOfItem" feature

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

File: data/pc/common/features.json [hunk 1]
```diff
@@ -333,5 +333,31 @@
...
+        "value": 8,
+        "versions": ["1.17", "latest"]
+      },
+      {
+        "value": 7,
+        "versions": ["1.14", "1.16"]
+      },
+      {
+        "value": 6,
+        "versions": ["1.10", "1.13"]
```
Comment (rom1504) [inline data/pc/common/features.json:351]: Why exclude 1.13.2 ?

Comment (u9g) [reply data/pc/common/features.json]: Fixed!

File: data/pc/common/features.json [hunk 2]
```diff
@@ -333,5 +333,31 @@
...
+    "name": "metadataIxOfItem",
+    "description": "item.metadata[this_ix] will be the item that was dropped on the ground",
+    "values": [
+      {
+        "value": 8,
+        "versions": ["1.17", "latest"]
+      },
+      {
+        "value": 7,
+        "versions": ["1.14", "1.16"]
```
Comment (rom1504) [inline data/pc/common/features.json:347]: are you sure you don't mean like 1.14_major to 1.16_major ?

Comment (u9g) [reply data/pc/common/features.json]: yep, fixed

**Pull request merged**